etn-ccis/blui-react-workflows
# Fix account details in-line error handling

Fixes blui-4284.



#### Changes proposed in this Pull Request:

- Fix account details in-line error handling
  - Only display error after initial onBlur
  - maintain error state when navigating between screens in the workflow



#### Screenshots / Screen Recording (if applicable)



#### To Test:

- update verify code screen component line 50 and 57 to `if (name?.length > 2)` to see errors (note: after this update the error message will be wrong as we are by default returning an error message that says you need at least 1 character.)
- yarn start:example
- click create account & continue through the workflow
- when you reach the account details screen add invalid first name and last name & blur (error should display)
- nav back to the previous screen and forward again to the account details screen
- verify error displays



#### Any specific feedback you are looking for?

-
